### ISSUE TITLE
[Prepropulated AI labels] The AI Label examples on inputs are not prepopulated in storybook ☂️

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#17419]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

Storybook https://react.carbondesignsystem.com/?path=/story/components-textinput--with-ai-label

### React version

Storybook version

### Automated testing tool and ruleset

n/a

### Assistive technology

n/a

### Description

I've put this under "Accessibility" because of the ramifications of the bug, but maybe it's likely better just listed as a bug...?

The prior experimental versions of the AI slug had demos for various inputs. These allowed the user to see how a pre-populated value in an input, flagged by the AI slug (now called AI Label), when altered, provided a revert function. This [revert function is covered in the component documentation](https://carbondesignsystem.com/components/ai-label/usage/#revert-function)

AFAIK, none of the implementations of the AI label that have been added to various inputs possess this ability in storybook, because none of them are prepopulated with values.

First, it does not make logical sense for the AI label to reside in an unpopulated field. The AI label is meant to indicate when a value has been generated by AI. WIth no value, the AI label should not exist.
<img width="1505" alt="image" src="https://github.com/user-attachments/assets/906869b9-20e2-48cb-a00a-b365a1a8531e">

Further, because the AI label is already in place on the empty field, I cannot trigger the revert function, even if I add a value to the field.

The result is one cannot test these interactions for accessibility, or use them to confirm expected behaviour.

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-textinput--with-ai-label

### Steps to reproduce

1. Go to any of the inputs in Storybook and choose "With AI label" from its variants. 
The AI label exists, but there is not value.
3. Populate a value.
4. No revert function appears.

### Suggested Severity

None

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by mbgower** [comment]
@aagonzales I mentioned this in some thread, but since it came up as a problem again today, i thought I'd better generate an issue for it.

**Comment by alina-jacob** [comment]
Thanks for creating and detailing out the issue Michael, just sharing some observations!

In the initial storybook examples, the Revert functionality existed only for two components, Text and Number input. This was also placed under a folder named Revert test. So technically, the other components were never built to show the full workflow from AI presence to Revert. But you're right, its worth adding them so users can fully understand the interaction. 

I did find [Revert available for the Form component](https://react.carbondesignsystem.com/?path=/story/components-form--with-ai-label&args=revertActive:!true&globals=theme:white), but its not working as intended.

**Comment by mbgower** [comment]
Somewhat related to https://github.com/carbon-design-system/carbon/pull/18405 (as per the link I gave in my prior comment.

**Comment by thyhmdo** [comment]
- Seems like we'd do an audit with all components by filtering out them "With AI label" from their variants.
- Add values (maybe dummy text) to those input fields 
- Add the revert function
- Consider those specific use cases, like Alina mentioned, for the Revert functionality, Text, Number input, and Form.
